app/controllers/api/public/v1/application_controller.rb

```diff
@@ -28,7 +28,7 @@ def default_per_page
         end
 
         def max_per_page
-          100
+          500
```

Comment:
https://api.publish-teacher-training-courses.service.gov.uk/api-reference.html#schema-pagination

Can this be updated as well please.